### PR TITLE
translation(burger-queen): Translate project README.md to pt-BR

### PR DESCRIPTION
**Pull request by paclaraujo**

### CODE REVIEW

File: projects/04-burger-queen/README.pt-BR.md [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+tamanho e a aparência dos botões, a visibilidade do estado atual do pedido, etc.
+
+A aplicação deve seguir 80% ou mais das pontuações de Performance, Progressive
+Web App, Accessibility e Best Practices do Lighthouse.
+
+O aplicativo deve fazer uso de `npm-scripts` e ter scripts `start`, `test`,
+`build` e `deploy`, que são responsáveis por inicializar, rodar os testes,
+empacotar e fazer deploy do aplicativo, respectivamente.
+
+Os testes unitários devem cubrir um mínimo de 70% de _statements_, _functions_,
```
Comment (JulianaAmoasei) [inline projects/04-burger-queen/README.pt-BR.md]: cobrir

File: projects/04-burger-queen/README.pt-BR.md [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+* [Firebase](https://firebase.google.com/)
+* [Serverless Architectures - Martin
+  Fowler](https://www.martinfowler.com/articles/serverless.html)
+
+#### Cloud functions
+
+* [Cloud functions - Firebase
+  Docs](https://firebase.google.com/docs/functions/?hl=es-419)
+
+<!-- ## Checklist
```
Comment (juanjordan) [inline projects/04-burger-queen/README.pt-BR.md]: Hey @paclaraujo, I think we can remove these comments as they were commented out from the Spanish version 11 months ago with not so much context: https://github.com/Laboratoria/bootcamp/commit/e20b6233869ed8e816f2dceab77af357bb294b25#diff-30a449096eab66b431fd16b28474f9e4R392

Comment (paclaraujo) [reply projects/04-burger-queen/README.pt-BR.md]: Done!

Comment (juanjordan) [reply projects/04-burger-queen/README.pt-BR.md]: I meant from the BR version too!

Comment (paclaraujo) [reply projects/04-burger-queen/README.pt-BR.md]: hahahahahaha sooo sorry. But know I hope it's ok !

**Pull request merged**